samples/openthread/ncp/prj.conf

```diff
@@ -99,4 +98,3 @@ CONFIG_SETTINGS=y
 
 # mbedTLS settings
 CONFIG_MBEDTLS_SSL_MAX_CONTENT_LEN=768
```

Comment:
Shouldn't this be removed as well?

Reply:
Let's leave it for now as nrf52833 does not use nrf_security at all now and the default is too big in mbedTLS. We would need to create a separate proj.conf overlay file.